extension/azure/test/test_files/azure.test

```diff
@@ -44,3 +47,21 @@ Alice|Bob
 Bob|Dan
 Bob|Carol
 Bob|Alice
+-STATEMENT COPY Knows FROM (LOAD FROM "abfss://kuzu-test/dirA/dirB/eKnows_2.csv" (file_format='azure') RETURN *);
```

Comment:
Not related to this PR. but I'm confusing why is azure a file_format? and why do we need to explicitly specify it?